src/app/components/login/login.component.ts

```diff
@@ -5,6 +5,11 @@ import { UserAuthenticationService } from '../../services/user-authentication.se
 import { first } from 'rxjs/operators';
 import { AlertService } from '../../services/alert.service';
 
+const LOGIN_FORM_MESSAGES = {
```

Comment:
I didn't say anything before, so it's ok for now, but later these kinds of constants could be in enum format. (not mandatory for fixing now)